main/src/main/scala/sbt/internal/librarymanagement/UnsafeLibraryManagementCodec.scala

```diff
@@ -0,0 +1,17 @@
+package sbt.internal.librarymanagement
```

Comment:
I didn't quite get the reasonind behind this. To me, this utility is totally part of `librarymanagement` and should go here. The rest of the utilities in this package are very similar to this one. Can you elaborate?